frontend/integration-tests/protractor.conf.ts

```diff
@@ -158,10 +158,14 @@ export const config: Config = {
       'tests/operator-hub/operator-hub.scenario.ts',
       'tests/developer-catalog.scenario.ts',
       'tests/monitoring.scenario.ts',
+      'tests/devconsole/dev-perspective.scenario.ts',
```

Comment:
Because for the devconsole tests I've created a new directory named `devconsole` so that all the test for developer console will be present there

Reply:
Do you want me to rename the directory name?

Reply:
@christianvogt  What are your views on this?
According to @ppitonak it's not a devconsole-specific functionality so should not be in the devconsole directory. 
IMO nobody in the openshift-console team would run this test as this takes you to the developer console. So the tests should reside in devconsole folder.

Reply:
I agree that the perspective switcher test is devconsole specific and should be organized as such in the directory structure. WDYT @christianvogt ?

Reply:
Honestly not too concerned but I made the original comment because perspective is not a dev console concept. It may have originated from dev console and dev console happens to provide a perspective, but so does admin. Therefore it's a core functionality of the console.